content/news/core-monthly/2023/core-monthly-2023-11-01-2023-11-30.md

```diff
@@ -0,0 +1,420 @@
+
+Another thing worth mentioning is updated jQuery, but not only in front office or back office, it has been updated everywhere thanks to work from [@Hlavtox](https://github.com/Hlavtox). You can see all the details in the [Upgrade jquery to latest version, drop polyfills PR](https://github.com/PrestaShop/PrestaShop/pull/34382/). This is a big change that definitely can impact your modules, so please check it out and make sure your modules are compatible with the new version of jQuery. 
+
+The new major versions also allow us to change, or fix, behaviors that don't quite work as they should and needed such changes. An example? Today, when we add a product to the shopping cart, and it is only available in quantity of 1 item, after adding it to the cart we will see a message on the product card that says that the product is no longer available. This could cause confusion for shoppers, sometimes they would leave the store thinking that someone was faster than them and made a purchase earlier. Starting with PrestaShop 9, the whole thing will work differently. You'll have an option to [Inform customer about quantity in cart](https://github.com/PrestaShop/PrestaShop/pull/33931).
+
+Thanks to [@Hlavtox](https://github.com/Hlavtox), we also have some other interesting changes:
+- He is still working on a project to change the way product properties are fetched, so that instead of Product::getProductProperties we use lazy property loading thanks to ProductLazyArray. One part of this project is to [remove references to the `getProductProperties` method from where products are fetched](https://github.com/PrestaShop/PrestaShop/pull/34498). This should speed up all front office.
+- He [fixed the display of gifts in the shopping cart](https://github.com/PrestaShop/PrestaShop/pull/34464)
+
+As you can see, the new version is not just a cleanup and changes under the hood. It's also changes that will realistically improve the user experience, whether it's for customers who buy from stores on PrestaShop software, or for merchants.
+
+### Modules
+
+The first module release that should catch your attention is certainly [blockreassurance](https://github.com/PrestaShop/blockreassurance/). [New 5.1.4 version](https://github.com/PrestaShop/blockreassurance/releases/tag/v5.1.4) fixes some security bugs so don't forget to update.
+
+Another noteworthy module is [the official integration with Google Analytics 4](https://github.com/PrestaShop/ps_googleanalytics/). The module has undergone quite a few changes, [and version 5](https://github.com/PrestaShop/ps_googleanalytics/pull/155) is in the testing phase at the moment. 
+
+The [comments module](https://github.com/PrestaShop/productcomments/) has changed as well (although changes not yet published). The JavaScript code responsible for fetching them has been migrataed to vanilla JS, and instead of relying on jQuery [it now uses natively available fetch()](https://github.com/PrestaShop/productcomments/pull/189).
```

Comment:
I don't think I used vanilla JS there ;)

Reply:
@leemyongpakvn what do you mean? Do you mean there are some leftovers with jQuery? just for the sake of clarity, vanilla js = clean js

Reply:
@kpodemski I meant I just replace jQuery by pure JS. According to http://vanilla-js.com/, vanilla JS is another JS framework. I don't know `vanilla js = clean js`. If it is a convention here then let's keep it ;)

Reply:
@leemyongpakvn this website is just a joke, no worries 👍🏻